### PR TITLE
Update chroma.js to 2.4

### PR DESCRIPTION
**Pull request by jdanford**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/gka/chroma.js/releases/tag/v2.4.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## chroma-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/chroma-js@latest/))
was missing the following properties:
1. Color
1. version
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 62a548e144143aee462b15ca7180992d9fe5dcf5
</p>

File: types/chroma-js/chroma-js-tests.ts [hunk 1]
```diff
@@ -22,7 +22,9 @@ function test_chroma() {
     chroma.hsl(330, 1, 0.6);
```
Comment (CharlieZhuo) [inline types/chroma-js/chroma-js-tests.ts:22]: Please add some tests with color function called with 4 arguments:
chroma.hsl(330, 1, 0.6 , 0.5);
 // @ts-expect-error
chroma.hcl(50, 40, 40 ,'invalid alpha');

**Review by CharlieZhuo — CHANGES_REQUESTED** [review]
Could you please write some tests with color function called with 4 arguments for those with optional alpha,Thanks.

**Review by CharlieZhuo — APPROVED** [review]

**Comment by jdanford** [comment]
Ready to merge

**Pull request merged**